### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by piotrkramarz**
Add branch Authentication to the develop. Styled form of sign-in/sign-up.

### CODE REVIEW

File: src/Auth.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ class Auth extends Component {
         {
           this.props.auth.data === null ?
             <div>
```
Comment (wojtrawi) [inline src/Auth.js]: Please provide appropriate description or just leave 'Placeholder'. The app summary should contain an image and be a separate component.

File: src/SignIn.js [hunk 2]
```diff
@@ -23,73 +23,54 @@ class signIn extends Component {
...
+    render() {
+      return (
+        <Grid>
+          <Row className="show-grid">
+            <Col xs={12} md={8}>
+              <Form onSubmit={this.handleSubmit} horizontal>
+                  {
+                      this.props.auth.error && <p>{this.props.auth.error.message}</p>
+                  }
+                <FormGroup controlId="formHorizontalEmail">
```
Comment (wojtrawi) [inline src/SignIn.js]: Non-unique 'id'. Is 'id' required here at all???

File: src/SignIn.js [hunk 3]
```diff
@@ -23,73 +23,54 @@ class signIn extends Component {
...
-  //     </Grid>
-  //
-  //   )
-  // }
+    render() {
+      return (
+        <Grid>
+          <Row className="show-grid">
+            <Col xs={12} md={8}>
+              <Form onSubmit={this.handleSubmit} horizontal>
```
Comment (wojtrawi) [inline src/SignIn.js]: Limit form's width. It doesn't look good on wide devices.

File: src/SignIn.js [hunk 4]
```diff
@@ -23,73 +23,54 @@ class signIn extends Component {
...
-  //           </Form>
-  //         </Col>
-  //       </Row>
-  //     </Grid>
-  //
-  //   )
-  // }
+    render() {
+      return (
+        <Grid>
```
Comment (wojtrawi) [inline src/SignIn.js]: Confirm Bootstrap usage approval from @robert-jemiolo

File: src/SignUp.js [hunk 5]
```diff
@@ -1,60 +1,87 @@
...
+                                        type="password"
+                                        placeholder="Password"
+                                        name="password"
+                                        onChange={this.handleChange}
+                                    />
+                                </Col>
+                            </FormGroup>
+                            <FormGroup>
+                                <Col smOffset={2} sm={10}>
+                                    <Button type="submit" bsStyle="danger" bsSize="large" block>
```
Comment (wojtrawi) [inline src/SignUp.js]: Button color is inappropriate. Suggest blue for 'sign in' and green for 'sign up'.

File: src/SignUp.js [hunk 6]
```diff
@@ -1,60 +1,87 @@
...
+    handleSubmit = event => {
+        event.preventDefault()
+        this.props.signUp(
+            this.state.login,
+            this.state.password
+        )
+    }
+
+    render() {
+        return (
```
Comment (wojtrawi) [inline src/SignUp.js]: The content of render method is virtually the same as for Sign In component. Consider creating reuseable component AuthenticateForm and use it in both SignIn and SignUp component with appropriate props passed (submission handlers, conrtol value change handler).

File: src/SignUp.js (hunk 5 above)
Comment (piotrkramarz) [reply src/SignUp.js]: Colors adjusted.

File: src/SignIn.js (hunk 4 above)
Comment (piotrkramarz) [reply src/SignIn.js]: Bootstrap usage approved.

File: src/SignIn.js (hunk 2 above)
Comment (piotrkramarz) [reply src/SignIn.js]: done

File: src/SignUp.js (hunk 6 above)
Comment (wojtrawi) [reply src/SignUp.js]: This still has not been done. In case you need some help, see today's commits in React jfdz6 repo and feel free to ask questions.

**Pull request merged**